vpr/src/timing/read_sdc.cpp

```diff
@@ -911,18 +912,33 @@ class SdcParseCallback : public sdcparse::Callback {
 
             bool found = false;
             for (tatum::DomainId domain : tc_.clock_domains()) {
-                const std::string& clock_name = tc_.clock_domain_name(domain);
-                if (std::regex_match(clock_name, clock_regex)) {
-                    found = true;
+                const auto& clock_name = tc_.clock_domain_name(domain);
+
+                if (tc_.is_virtual_clock(domain)) {
```

Comment:
Need a comment here I think saying what the overall idea is.  

If virtual_clock must match the clock_regex.  Otherwise, look for all aliases of the clock_name (nets that passed the clock signal through?) and see if any of them matches the clock_regex.  A comment saying why this is the right logic (and likely an example) would be helpful.